transactions/0074.md

```diff
@@ -0,0 +1,303 @@
+```
+
+### Bytewise Breakdown
+```javascript
+fe8a6a0c00 // block height VarInt
+0b // 1 byte tree height (eleven)
+01 // nLeaves VarInt at level 11
+00 // offset VarInt
+01 // data follows
+af8764ce7e1cc132ab5ed2229a005c87201c9a5ee15c0f91dd53eff31ab30cd4 // leaf at 11th level of tree
+01 // nLeaves VarInt at level 10
+03 // offset VarInt
+00 // no data follows
+01 // nLeaves VarInt at level 9
+04 // offset VarInt
+01 // data follows
+1ccd9f8f64f4d0489b30cc815351cf425e0e78ad79a589350e4341ac165dbe45 // leaf at 9th level of tree
+01 // nLeaves VarInt at level 8
+0a // offset VarInt
+01 // data follows
+502e59ac92f46543c23006bff855d96f5e648043f0fb87a7a5949e6a9bebae43 // leaf at 8th level of tree
+01 // nLeaves VarInt at level 7
+16 // offset VarInt
+01 // data follows
+8120cafa17309c0bb0e0ffce835286b3a2dcae48e4497ae2d2b7ced4f051507d // leaf at 7th level of tree
+01 // nLeaves VarInt at level 6
+2e // offset VarInt
+01 // data follows
+e07a2bb8bb75e5accff266022e1e5e6e7b4d6d943a04faadcf2ab4a22f796ff3 // leaf at 6th level of tree
+01 // nLeaves VarInt at level 5
+5e // offset VarInt
+01 // data follows
+5881826eb6973c54003a02118fe270f03d46d02681c8bc71cd44c613e86302f8 // leaf at 5th level of tree
+01 // nLeaves VarInt at level 4
+bf // offset VarInt
+00 // no data follows
+01 // nLeaves VarInt at level 3
+fd7c01 // offset VarInt (three hundred and eighty)
+01 // data follows
+93b3efca9b77ddec914f8effac691ecb54e2c81d0ab81cbc4c4b93befe418e85 // leaf at 3th level of tree
+01 // nLeaves VarInt at level 2
+fdfb02 // offset VarInt
+00 // data follows
+02 // nLeaves VarInt at level 1
+fdf405 // offset VarInt
+01 // data follows
+0671394f72237d08a4277f4435e5b6edf7adc272f25effef27cdfe805ce71a81 // leaf 1 at 1st level of tree
+fdf505 // offset VarInt
+01 // data follows
+262bccabec6c4af3ed00cc7a7414edea9c5efa92fb8623dd6160a001450a5282 // leaf 2 at 1st level of tree
+04 // nLeaves VarInt at level 0
+fde80b // offset VarInt
+01 // data follows
+11774f01d26412f0d16ea3f0447be0b5ebec67b0782e321a7a01cbdf7f734e30 // leaf 1 at 0 level of tree
+fde90b // offset VarInt
+01 // data follows
+004e53753e3fe4667073063a17987292cfdea278824e9888e52180581d7188d8 // leaf 2 at 0 level of tree
+fdea0b // offset VarInt
+01 // data follows
+5e441996fc53f0191d649e68a200e752fb5f39e0d5617083408fa179ddc5c998 // leaf 3 at 0 level of tree
+fdeb0b // offset VarInt
+00 // no data follows
+```
+
+## JSON Encoding
+
+In the JSON encoding - we start with a height for the whole BUMP. A path Array index corresponds to the height within the Merkle tree, so we start with level 0 which is the txids themselves. Within each array element we contain one or more leaves which are specified as an offset number as the key and a leaf as the value.  
+
+`{ [offset]: leaf }` This is to aid in the speed of merkle root calculation, since the offset is given, and so can be used to lookup the corresponding value directly rather than having to iterate through all leaves.  
```

Comment:
I like that approach with switching offset and leaf places it this format, comparing to the Compound Merkle Path.